helm/values-dev.yaml

```diff
@@ -127,6 +128,7 @@ microservice-chart:
     NPG_READ_TIMEOUT: "10000"
     NPG_CONNECTION_TIMEOUT: "10000"
     NPG_CARDS_PSP_LIST: "BCITITMM,CIPBITMM,BIC36019,UNCRITMM,BPPIITRRXXX,PPAYITR1XXX"
+    NPG_AUTHORIZATION_REQUEST_TIMEOUT: "60"
```

Comment:
what about explicit this timeout measure unit in env variable name? seconds? millis?